python/Makefile

```
@@ -60,6 +80,142 @@ help:
 	@echo "  - All DBR versions use Java 8 to match Databricks Runtime"
 	@echo ""
 
+# Setup development environment
+.PHONY: setup
+setup:
+	@echo "Setting up development environment..."
+	@if [ -f .env.versions ]; then \
+		. .env.versions && \
+		python -m pip install --upgrade pip --quiet && \
+		python -m pip install hatch==$${HATCH_VERSION} "$${CLICK_CONSTRAINT}" --quiet && \
+		echo "Development tools installed successfully"; \
+	else \
+		echo "ERROR: .env.versions file not found"; \
+		exit 1; \
+	fi
+
+# Complete initialization for new developers - one command to set up everything
+.PHONY: init
+init:
+	@echo " Initializing complete development environment..."
+	@echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+	@$(MAKE) setup
+	@echo ""
+	@$(MAKE) check-env
+	@echo ""
+	@echo " Creating default virtual environment..."
+	@$(MAKE) venv
+	@echo ""
+	@echo " Running smoke test..."
+	@$(MAKE) lint 2>&1 | head -5
+	@echo ""
+	@echo " Development environment initialization complete!"
+	@echo ""
+	@echo " Next steps:"
+	@echo "  1. Review CONTRIBUTING.md for detailed documentation"
+	@echo "  2. Run 'make test' to run tests"
+	@echo "  3. Run 'make help' to see all available commands"
+
+# Quick status check
+.PHONY: status
+status:
+	@# Check development tools
+	@if [ -f .env.versions ]; then \
+		. .env.versions && \
+		if command -v hatch >/dev/null 2>&1; then \
+			HATCH_VER=$$(hatch --version | grep -oE '[0-9]+\.[0-9]+\.[0-9]+'); \
+			if [ "$${HATCH_VER}" = "$${HATCH_VERSION}" ] && python -c "import click; exit(0 if tuple(map(int, click.__version__.split('.'))) < (8, 3, 0) else 1)" 2>/dev/null; then \
+				echo "Development tools: OK"; \
+			else \
+				echo "Development tools: needs update (run 'make setup')"; \
+			fi; \
+		else \
+			echo "Development tools: not installed (run 'make setup')"; \
+		fi; \
+	else \
+		echo "Development tools: config missing"; \
+	fi
+	@# Check Java
+	@if [ -d "$(JAVA_HOME_TARGET)" ]; then \
+		echo "Java 8: OK"; \
+	else \
+		echo "Java 8: not installed"; \
+	fi
+	@# Check virtual environments
+	@if [ -d ".venv" ]; then \
+		COUNT=$$(ls -1 .venv | wc -l | tr -d ' '); \
```

can we use the variable `${VENV_DIR}` here instead of hardcoding
```suggestion
	@if [ -d "${VENV_DIR}" ]; then \
		COUNT=$$(ls -1 ${VENV_DIR} | wc -l | tr -d ' '); \
```